frontend/js/src/home/NumberCounter.tsx

```diff
@@ -6,13 +6,11 @@ type NumberCounterProps = {
 
 function NumberCounter({ count }: NumberCounterProps) {
   const countString = count?.toString();
-  const reversedStr = countString?.split("").reverse().join("");
-  const groupsOfThree = reversedStr?.match(/\d{1,3}/g);
-  const reversedGroupsOfCount = groupsOfThree ? groupsOfThree.reverse() : [];
+  const groupsOfThree = countString?.match(/\d{1,3}/g) ?? [];
```

Comment:
This does not work well if you have the count = `1234567890`. It breaks it into `['123', '456', '789', '0']`. 

Instead you can try out this new regex which breaks it into `['1', '234', '567', '890']`

```suggestion
  const groupsOfThree = countString?.match(/(\d+?)(?=(\d{3})+(?!\d)|$)/g) ?? [];
```